pages/common/gh-release.md

```diff
@@ -9,20 +9,20 @@
 
 - Display information about a specific release:
 
-`gh release view {{tag}}`
+`gh release view "{{tag}}"`
 
 - Create a new release:
 
-`gh release create {{tag}}`
+`gh release create "{{tag}}"`
 
 - Delete a specific release:
 
-`gh release delete {{tag}}`
+`gh release delete "{{tag}}"`
 
 - Download assets from a specific release:
 
-`gh release download {{tag}}`
+`gh release download "{{tag}}"`
 
 - Upload assets to a specific release:
 
-`gh release upload {{tag}} {{path/to/file1}} {{path/to/file2}}`
+`gh release upload "{{tag}}" {{path/to/file1 path/to/file2 ...(1..)}}`
```

Comment:
What means `...(1..)`?
```suggestion
`gh release upload "{{tag}}" {{path/to/file1 path/to/file2 ...}}`
```
Why tldr is adding Calculus notation? :laughing: these pages should be easy to read for non-college people. We already have a lot of formal language used to explain examples.

Reply:
> What means `...(1..)`?
> ```suggestion
> `gh release upload "{{tag}}" {{path/to/file1 path/to/file2 ...}}`
> ```
> Why tldr is adding Calculus notation? :laughing: these pages should be easy to read for non-college people. We already have a lot of formal language used to explain examples.

Exactly, I was confused too.

Reply:
> Why tldr is adding Calculus notation? laughing these pages should be easy to read for non-college people. We already have a lot of formal language used to explain examples.

It's a formalized syntax for ellipsis that supports ranges. In this case it means to expect 1 or more files here. As now there is no standardized way to present how much arguments are expected to ellipsis.

Reply:
The user should be able to read pages without learn a syntax guide. I'm not able to understand `...(n...)` without read the guide.

Reply:
I think I'll change this syntax to another one as it causes confusion as I got.

Reply:
Please rereview this line again. :)